test_specs.sh

```
@@ -138,10 +163,20 @@ while IFS= read -r -d "" file; do
     fi
 done < <(find "$specs_dir" -type f -name "*.xml" -print0)
 
+while IFS= read -r -d "" file; do
+    rel_path="${file#$specs_dir/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)